webapp/src/components/admin_settings/jitsi_settings.tsx

```diff
@@ -0,0 +1,246 @@
+import React, {useCallback, useEffect, useState, useMemo, ChangeEvent} from 'react';
+import {FormattedMessage} from 'react-intl';
+import JaaSSection from './jaas_section';
+import JitsiSection, {JITSI_NAMING_SCHEME} from './jitsi_section';
+import {id as pluginId} from '../../manifest';
+import I18nProvider from 'components/i18n_provider';
+import {AdminConfig} from 'mattermost-redux/types/config';
+
+type Props = {
+    id: string,
+    label: string,
+    value: Settings,
+    disabled: boolean,
+    config: AdminConfig,
+    license: object,
+    setByEnv: boolean,
+    onChange: Function,
+    registerSaveAction: Function,
+    setSaveNeeded: Function,
+    unRegisterSaveAction: Function
+};
+
+type Settings = {
+    usejaas?: boolean,
```

Comment:
i think every other places you named it like this
```suggestion
    useJaaS?: boolean,
```